Week1/index.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const fs = require('fs');
+const mysql = require('mysql');
+const { promisify } = require('util');
+const readFile = promisify(fs.readFile);
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+});
+
+const executeQuery = promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  const CREATE_DATABASE = `CREATE DATABASE IF NOT EXISTS world`;
+  const USE_DATABASE = `USE world`;
+
+  const CREATE_CITIES_TABLE = `
+  CREATE TABLE IF NOT EXISTS cities (
+    id INT NOT NULL AUTO_INCREMENT, 
+    city_name VARCHAR(254), 
+    population INT, 
+    district VARCHAR(254), 
+    country_code VARCHAR(3), 
+    primary key(id)
+    );`;
+  const CREATE_COUNTRIES_TABLE = `
+  CREATE TABLE IF NOT EXISTS countries (
+    id INT NOT NULL AUTO_INCREMENT,
+    country_name VARCHAR(254),
+    capital VARCHAR(254),
+    local_name VARCHAR(254),
+    indep_year INT,
+    government_form VARCHAR(254),
+    population INT,
+    head_of_state VARCHAR(254),
+    region VARCHAR(254),
+    continent ENUM('Africa', 'Antarctica', 'Asia', 'Australia','Oceania', 'Europe', 'North America', 'South America' ),
+    surface_area FLOAT,
+    life_expectancy FLOAT,
+    GNP FLOAT,
+    GNP_old FLOAT,
+    primary key(id)
+    );`;
+
+  connection.connect(err => {
+    if (err) {
+      console.log('Error while connecting to server');
+    } else {
+      console.log('Successfully connected to server');
+    }
+    process.exit;
+  });
+
+  try {
+    await executeQuery(CREATE_DATABASE);
+    await executeQuery(USE_DATABASE);
+    await executeQuery(CREATE_CITIES_TABLE);
+    await executeQuery(CREATE_COUNTRIES_TABLE);
+    const { cities, countries } = JSON.parse(await readFile('./data.json', 'utf8'));
+    cities.forEach(async city => {
+      try {
+        await executeQuery(`REPLACE INTO cities SET city_name='${city.name}', population=${city.population},
```

Comment:
I would not use replace, it is hard to understand what it will actually do with the data. But it is fine for testing purpose.